packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -373,19 +374,24 @@ class ScaleTransition extends AnimatedWidget {
   /// an alignment of (0.0, 1.0).
   final Alignment alignment;
 
+  /// The filter quality with which to apply the transform as a bitmap operation.
+  ///
+  /// The transform will be applied by re-rendering the child if this value is null,
+  /// otherwise it controls the quality of the bitmap transform applied to a bitmap
+  /// rendering of the child.
```

Comment:
Consider using `{@template ...}` and `{@macro ...}` to avoid repeating the same text. This will reduce the chance that the doc comments will deviate from each other.

Example: https://github.com/flutter/flutter/blob/88809aa24720705d1cd8de0bfc46faf932fe0b6e/packages/flutter/lib/src/widgets/pages.dart#L18

Reply:
Done